front_end/src/components/landing_page/search_widget/CalendarPicker.js

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from "react";
+import ReactDOM from "react-dom";
+import { withStyles } from "@material-ui/core/styles";
+
+import { PropTypes } from "prop-types";
+import { connect } from "react-redux";
+import { Form, Input, FormGroup, Container, Label } from "reactstrap";
+import "react-dates/lib/css/_datepicker.css";
+import "react-dates/initialize";
+import "./CalendarStyleOverride.css";
+
+import { DateRangePicker, DayPickerRangeController } from "react-dates";
+
+const styles = theme => ({
+  root: {
+    // display: "flex",
+    // flexWrap: "wrap",
+    padding: theme.spacing.unit
+  },
+  supportText: {
+    top: -10
+  }
+});
+
+class CalendarPicker extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      date: null,
+      focused: null,
+      name: "hai",
+      labelWidth: 10
+    };
+  }
+
+  handleChange = event => {
+    this.setState({ [event.target.name]: event.target.value });
+  };
+
+  render() {
+    const { classes } = this.props;
+
+    return (
+      <div className={classes.root}>
+        <DateRangePicker
+          startDate={this.state.startDate} // momentPropTypes.momentObj or null,
+          startDateId="your_unique_start_date_id" // PropTypes.string.isRequired,
+          endDate={this.state.endDate} // momentPropTypes.momentObj or null,
+          endDateId="your_unique_end_date_id" // PropTypes.string.isRequired,
+          onDatesChange={({ startDate, endDate }) =>
+            this.setState({ startDate, endDate })
+          } // PropTypes.func.isRequired,
```

Comment:
Unnecessary comment.

PropTypes are defined outside the class definition.

Reply:
It's a comment I took over by Airbnb... I wasn't sure what they were trying to tell us here , but I left it since I don't know what they mean